Microsoft.Toolkit.Uwp/Helpers/BackgroundTaskHelper.cs

```diff
@@ -43,6 +43,12 @@ public static bool IsBackgroundTaskRegistered(Type backgroundTaskType)
         /// <returns>Background Task that was registered with the system</returns>
         public static async Task<BackgroundTaskRegistration> RegisterAsync(Type backgroundTaskType, IBackgroundTrigger trigger, bool enforceConditions = true, params IBackgroundCondition[] conditions)
         {
+            // Check if the task is already registered.
```

Comment:
Seem that I misunderstood some document. And whenever I try to register the BackgroundTask the second time I get an System.Exception: Not enough quota is available to process this command.